src/db-adapters/Mongoose/MongooseAdapter.js

```diff
@@ -614,4 +596,29 @@ export default class MongooseAdapter {
 
     return words.join(" ");
   }
+
+  static getIdQueryType(idOrIds) {
```

Comment:
This is looking good! Before I merge it, can you please add unit tests for this new function in `/test/unit/db-adapters/MongooseAdapter.js`; all the static methods are tested there, and I think this one should be too.